node_modules/col-core/lib/db.js

```diff
@@ -468,6 +468,22 @@ var setUpModel = function(sequelize) {
       'type': Sequelize.INTEGER,
       'defaultValue': 0
     },
+    'impact_percentile': {
+      'type': Sequelize.INTEGER,
+      'defaultValue': 0
+    },
+    'impact_score': {
+      'type': Sequelize.INTEGER,
+      'defaultValue': 0
+    },
+    'trending_percentile': {
+      'type': Sequelize.INTEGER,
+      'defaultValue': 0
+    },
+    'trending_score': {
+      'type': Sequelize.INTEGER,
+      'defaultValue': 0
+    },
```

Comment:
@johncrossman Good question; existing code is inconsistent on that point. But I've amended the new columns to make it explicit.